docs/samples/custom/grpc-server/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 The Knative Authors
```

Comment:
update to "2020 The Kubeflow Authors" ?

Reply:
I have updated the sample code to a simple helloworld server.